ocdskingfisherarchive/database_process.py

```diff
@@ -0,0 +1,31 @@
+import sqlalchemy as sa
```

Comment:
Don't use SQLAlchemy. Use psycopg2, like in Kingfisher Views: https://ocp-software-handbook.readthedocs.io/en/latest/python/code.html#preferred-packages